deployment/CP12WESW_Deploy.csv

```diff
@@ -1,23 +1,23 @@
+AR82,,AR87,,CP12WESW-SBS01-00-STCENG000,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,,CGCON-BSWSTC-00001,35.9545,-75.3334,,0,31,,,A
+AR82,,AR87,,CP12WESW-SBS01-05-MOPAK0000,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,,CGINS-MOPAK0-43252,35.9545,-75.3334,,0,31,,,A
+AR82,,AR87,,CP12WESW-SBI01-02-CTDMOS011,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,,CGINS-CTDMOS-12304,35.9545,-75.3334,,1,31,,,A
+AR82,,AR87,,CP12WESW-SBS01-03-VELPTA000,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,,CGINS-VELPTA-12224,35.9545,-75.3334,,1,31,,,A
+AR82,,AR87,,CP12WESW-WFP01-00-WFPENG001,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,CGVEH-CWFPCP-03501,CGCON-CWFPCP-03501,35.9545,-75.3334,,17,31,,,A
+AR82,,AR87,,CP12WESW-WFP01-01-CTDPFP000,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,CGVEH-CWFPCP-03501,CGINS-CTDPFP-00070,35.9545,-75.3334,,17,31,,,A
+AR82,,AR87,,CP12WESW-WFP01-02-DOFSTP000,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,CGVEH-CWFPCP-03501,CGINS-DOFSTP-04320,35.9545,-75.3334,,17,31,,,A
+#AR82,,AR87,,CP12WESW-WFP01-03-FLORTP000,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,CGVEH-CWFPCP-03501,,35.9545,-75.3334,,17,31,not installed on prawler for first deployment,,A
+AR82,,AR87,,CP12WESW-MFI01-02-CTDMOS031,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,,CGINS-CTDMOS-12337,35.9545,-75.3334,,30,31,,,A
+AR82,,AR87,,CP12WESW-MFI01-02-ADCPTQ010,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,,CGINS-ADCPTQ-01036,35.9545,-75.3334,,30,31,,,A
+AR82,,AR87,,CP12WESW-MFI01-02-WAVSSQ010,1,1,2024-04-15T14:23:00,2025-04-12T23:03:00,CGMCP-12WESW-00001,,CGINS-WAVSSQ-01036,35.9545,-75.3334,,30,31,,,A
+AR87,,,,CP12WESW-SBS01-00-STCENG000,2,1,2025-04-12T20:32:00,,CGMCP-12WESW-00002,,CGCON-BSWSTC-00004,35.94552,-75.33332,,0,32,,,B
+AR87,,,,CP12WESW-SBS01-05-MOPAK0000,2,1,2025-04-12T20:32:00,,CGMCP-12WESW-00002,,CGINS-MOPAK0-99998,35.94552,-75.33332,,0,32,,,B
+AR87,,,,CP12WESW-SBI01-02-CTDMOS011,2,1,2025-04-12T20:32:00,,CGMCP-12WESW-00002,,CGINS-CTDMOS-13576,35.94552,-75.33332,,1,32,,,B
+AR87,,,,CP12WESW-SBS01-03-ADCPUP000,2,1,2025-04-12T20:32:00,,CGMCP-12WESW-00002,,CGINS-ADCPUP-00277,35.94552,-75.33332,,1,32,,,B
+AR87,,,,CP12WESW-WFP01-00-WFPENG001,2,1,2025-04-12T20:32:00,,CGMCP-12WESW-00002,CGVEH-CWFPCP-03504,CGCON-CWFPCP-03504,35.94552,-75.33332,,22,32,,,B
+AR87,,,,CP12WESW-WFP01-01-CTDPFP000,2,1,2025-04-12T20:32:00,,CGMCP-12WESW-00002,CGVEH-CWFPCP-03504,CGINS-CTDPFP-00066,35.94552,-75.33332,,22,32,,,B
+AR87,,,,CP12WESW-WFP01-02-DOFSTP000,2,1,2025-04-12T20:32:00,,CGMCP-12WESW-00002,CGVEH-CWFPCP-03504,CGINS-DOFSTP-04318,35.94552,-75.33332,,22,32,,,B
+AR87,,,,CP12WESW-WFP01-03-FLORTP000,2,1,2025-04-12T20:32:00,,CGMCP-12WESW-00002,CGVEH-CWFPCP-03504,CGINS-FLORTP-08569,35.94552,-75.33332,,22,32,,,B
```

Comment:
Per CP12WESW-00002 the Prawler lower depth limit is 21 m.

Reply:
latest deck dwg (rev C) is 2-22 m interval. Slacked that to you as well. Config tab probably needs updating.